src/common/tvgBezier.cpp

```diff
@@ -41,14 +41,75 @@ static float _lineLength(const Point& pt1, const Point& pt2)
 }
 
 
+static float _lineExactLength(const Point& pt1, const Point& pt2)
+{
+    Point diff = {pt2.x - pt1.x, pt2.y - pt1.y};
+    return sqrtf(diff.x * diff.x + diff.y * diff.y);
```

Comment:
We only need a single line for this fix. I think you are going overboard with the changes?